### PR TITLE
#120: API-containers-Bitset from md to rst

### PR DESCRIPTION
**Pull request by marcinwrobel1986**
### THIS PR:
- API/containers/Bitset transition from .md to .rst

[Link to the current page](https://kokkos.github.io/kokkos-core-wiki/API/containers/Bitset.html)

Below render of the API/containers/Bitset after transition:
Zrzut ekranu z 2022-08-30 16-42-59
Zrzut ekranu z 2022-08-30 16-43-16
Zrzut ekranu z 2022-08-30 16-43-31

Zrzut ekranu z 2022-08-30 16-46-36
Zrzut ekranu z 2022-08-30 16-46-57
Zrzut ekranu z 2022-08-30 16-47-11

### CODE REVIEW

**Comment by fnrizzi** [comment]
@marcinwrobel1986 is this just the new render with dark/light mode? where is the old one?

**Comment by marcinwrobel1986** [comment]
> @marcinwrobel1986 is this just the new render with dark/light mode? where is the old one?

The old one is [HERE](https://kokkos.github.io/kokkos-core-wiki/API/containers/Bitset.html)

File: docs/source/API/containers/Bitset.rst [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+Header file: ``Kokkos_Bitset.hpp``
+
+Class Interface
+---------------
+
+.. cpp:class:: template <typename Device> Bitset
+
+  :cpp:`Kokkos::Bitset` represents a thread safe view to a fixed-size (at run-time) sequence of N bits.
+
```
Comment (crtrott) [inline docs/source/API/containers/Bitset.rst:15]: Remove the ConstBitset here.

**Review by crtrott — CHANGES_REQUESTED** [review]
Just remove that ConstBitset thing further up then its good.

File: docs/source/API/containers/Bitset.rst (hunk 1 above)
Comment (fnrizzi) [reply docs/source/API/containers/Bitset.rst:15]: done

**Pull request merged**